mlx/traceability.py

```diff
@@ -170,6 +170,8 @@ def process_item_nodes(app, doctree, fromdocname):
     Replace all ItemList nodes with a list of the collected items.
     Augment each item with a backlink to the original location.
 
+    If the ``checklist_item_regex`` is configured, a warning is reported
```

Comment:
This is better added to documentation part...

Reply:
Right, I forgot to add it in there. Done here: 48b4813c31a2e21e82826d0e3a97c26c8b14f67f